### PR TITLE
Cleaner Dark Theme Script

### PR DESCRIPTION
**Pull request by WilsontheWolf**

### CODE REVIEW

File: src/consts.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const server = require('../server');
 
 module.exports.dlocklyGuild = function () {
-  return server.bot.guilds.get('591692042304880815');
+  return server.bot.guilds.get(process.env.DISCORD_GUILD_ID);
```
Comment (WilsontheWolf) [inline src/consts.js]: accidentally left this in but it would be convenient for me

File: www/html/partials/footer.ejs [hunk 2]
```diff
@@ -5,12 +5,11 @@
 
 <script>
   let check = document.getElementById('darkCheckbox');
-  let css = document.getElementById('darkCSS')
   check.checked = getCookie('darkTheme') == 'true';
   changeTheme();
   
   function changeTheme() {
-    css.disabled = !check.checked;
+    document.getElementById('darkCSS').disabled = !check.checked;
```
Comment (WilsontheWolf) [inline www/html/partials/footer.ejs]: this was my commit why not just close it?

Comment (Alexejhero) [reply www/html/partials/footer.ejs]: https://github.com/WilsontheWolf/Dlockly/blob/2eb1e2ab2a7e003a2ed048911dfd58a868864c23/src/consts.js#L4

**Comment by WilsontheWolf** [comment]
u deleted my branch

**Comment by Alexejhero** [comment]
I didn't, the repo is set to automatically delete head branches, I'll disable that.

**Comment by WilsontheWolf** [comment]
:thonk: 
image

**Comment by Alexejhero** [comment]
In case you're not aware, the person who merges the pull request automatically does all of the following things:
- Deletes the head branch if the option is enabled in the repo settings
- If the PR has been merged into the master branch, closes all issues and pull requests that have been mentioned with the `fixes` or `closes` prefix
- Author the merge commit
- Becomes a 'participant' in the pull request

**Comment by Alexejhero** [comment]
image

**Pull request merged**